source/iot/Mqtt5Client.cpp

```diff
@@ -408,9 +408,15 @@ namespace Aws
             return *this;
         }
 
-        Mqtt5ClientBuilder &Mqtt5ClientBuilder::WithAckTimeoutSeconds(uint32_t ackTimeoutSeconds) noexcept
+        Mqtt5ClientBuilder &Mqtt5ClientBuilder::WithAckTimeoutSec(uint32_t ackTimeoutSec) noexcept
         {
-            m_options->WithAckTimeoutSeconds(ackTimeoutSeconds);
+            m_options->WithAckTimeoutSec(ackTimeoutSec);
+            return *this;
+        }
```

Comment:
Even though it's trivial, let's use a rule that the deprecated function should always just call the non-deprecated variant rather than repeat what the non-deprecated variant does.